lambdas/file_access_patterns/lambda_for_fileUpload/index.js

```diff
@@ -31,31 +49,31 @@ exports.handler = async (event) => {
     f_type: reqBody.f_type,
   };
 
-  try{
-    await dynamo.put({
-      TableName:"V-Transfer",
-      Item: file
-    }).promise()
+  let generateUrlTask;
+  const params = { FunctionName: 'lambda_for_generateURL', InvocationType: 'RequestResponse',
+    Payload: getGenerateUrlPayload(file.hash, file.SK.split('#')[1])
   }
-  catch(err){
-    return response(500,"Internal Server Error",undefined);
+  try {
+    const res = await Lambda.invoke(params).promise()
+    generateUrlTask = JSON.parse(res.Payload)
+  } catch(err) { return response(500, JSON.stringify(err), undefined) }
```

Comment:
Return `Internal Server Error` as the error message.

Reply:
updated please check. Thanks